docs/ocis/guides/migrate-data-rclone.md

```diff
@@ -0,0 +1,103 @@
+
+---
+title: "Migrate Data using rclone"
+date: 2020-06-12T14:35:00+01:00
+weight: 8
+geekdocRepo: https://github.com/owncloud/ocis
+geekdocEditPath: edit/master/docs/ocis/guides
+geekdocFilePath: migrate-data-rclone.md
+geekdocCollapseSection: true
+---
+
+People keep asking on how to migrate data from other cloud storage to Infinte Scale.
+
+There are too many cloud variants and use cases out there to have a migration path for all at hand, but let's see what we can start with: There is the famous swiss army knife for clouds called rclone avialable.
+
+The awesome rclone makes it easy to migrate data from one installation to another on a user per user base. A very good first step.
+
+This article explains by the example of Nextcloud how you would migrate your data from an running NC to Infinite Scale. A prerequisite is that you have Infinite Scale already set up and running on a different network address.
+
+{{< hint warning >}}
+Be prepared that migration can take some time. Also, check the size of your data. This example was around 1.5 GiB of data, that obviously went fast.
+
+And of course: Have a backup! Even if this method only reads from the source, you never know.
+{{< /hint >}}
+
+## Install rclone
+
+Check the [rclone website on how to install](https://rclone.org/install/) rclone.
+
+## Create Users
+
+First, decide on a user that you want to migrate. To create the user on Infinite Scale, log in as an admin user and create the desired user. Create and set groups accordingly.
+
+For the next step, you need the user credentials on both the source- and the destination cloud.
+
+## Configure rclone Remotes
+
+To be able to address the clouds in rclone, you need to configure so called _remotes_. It is nothing else than a shortcut for the combination of
+- which kind of cloud are you talking to
+- the URL
+- username
+- evtl. password
```

Comment:
eventual? what is "evtl"?

Reply:
`eventuell`, german for `maybe` / `perhaps`